mep/people/tests/test_forms.py

```diff
@@ -80,4 +82,31 @@ def test_init(self):
         assert peeps.last() not in mergeform.fields['primary_person'].queryset
 
 
+class TestRadioWithDisabled(TestCase):
```

Comment:
time to put these in a library/installable app soon-ish? on several projects now

Reply:
Open to that, but a little wary of ending up with a kitchen sink kind of app like [eulcommon](https://github.com/emory-libraries/eulcommon). Let's discuss as a team sometime.